packages/reactivity/__tests__/watch.spec.ts

```diff
@@ -277,4 +278,44 @@ describe('watch', () => {
 
     expect(dummy).toEqual([1, 2, 3])
   })
+
+  test('removing a watcher while stopping its effectScope', async () => {
```

Comment:
The test case might fit better in `packages/reactivity/__tests__/effectScope.spec.ts`

Reply:
done~